### ISSUE TITLE
Feature Request: Dot indicators to represent events on a date.

### ISSUE THREAD

**Issue by jameswwwebster** [richardtop/CalendarKit#95]
On the Apple calendar, on month view, each day can have a dot underneath the date to represent the presence or absence of an event. This isn't present on the day view on the Apple Calendar, however I think that it is something the top week-banner could benefit from and could be added to this library.

This is a feature that I plan to implement myself and submit as a pull request, however it would be good if there could be some feedback about potential pitfalls or suggested implementation before any work is done.

**Comment by richardtop** [comment]
It's definitely a good idea.
Few suggestions / possible solutions, that I have in mind:
1. It's possible to turn this feature on and off by setting a property in `DaySelectorStyle`.
2. The property, in its turn, will enable the feature in `DateLabel`.
3. All the changes in style will be propagated through `CalendarStyle` `NSCopying` protocol.

Another idea is to change `Header` to use `UICollectionView` or implement a design similar to it. So, for example, to register custom cells.

**Comment by vgyan** [comment]
Good One.. Waiting for this

**Comment by lincolntran** [comment]
@jameswwwebster  Are you still working on this?  I'm planning to implement it myself, but don't want to duplicate the work.

Thanks

**Comment by jameswwwebster** [comment]
@lincolntran, No, I never got the chance to start on this.

**Comment by lincolntran** [comment]
Ok, I'll do it then if you don't mind.

**Comment by jameswwwebster** [comment]
Of course! :) This is the brilliance of open source software, everybody can contribute.

**Comment by sinisa503** [comment]
Did you work something on this? I have implemented this feature in my project but I did not put this code here because It's not made for framework but project specific. Maybe I can get involved if you need...

**Comment by jameswwwebster** [comment]
No, I never did get round to working on this.

**Comment by ankitjaiswal1994** [comment]
@sinisa503 Could you please help us if you have already did this ?

**Comment by ysaucedo** [comment]
I need this functionality, I can cooperate with money who makes the adjustment

**Comment by ankitjaiswal1994** [comment]
@ysaucedo If you need something like below attachment I can help you without money.
![Simulator Screen Shot - iPhone 6 - 2019-09-10 at 19 11 26](https://user-images.githubusercontent.com/20721521/64618944-de862d00-d3fe-11e9-83ed-4c2b023a8a0f.png)
![Simulator Screen Shot - iPhone 6 - 2019-09-10 at 19 11 27](https://user-images.githubusercontent.com/20721521/64618952-e34ae100-d3fe-11e9-82bb-162f04005aac.png)

**Comment by ysaucedo** [comment]
@ankitjaiswal1994 

Hi! It's what i'm looking for,
Can you share the control you mention?

**Comment by ankitjaiswal1994** [comment]
@ysaucedo I tried in Calendarkit it doesnt seem possible. I was not able to get that view where I can create that dot.

So I used this library : https://github.com/WenchaoD/FSCalendar

With little bit customization it worked as the exact way what I need.

**Comment by MrCarter31** [comment]
Has anyone implemented this feature?

**Comment by richardtop** [comment]
@MrCarter31 no, feel free to propose a pull request implementing this feature.
